PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Pull Request Guidelines for Authors
+
+Summary of [RFC66 - Pull Request & Code Review Standards](https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards):
+
+<!-- Go over all the following points, and put an `x` in the boxes that are true -->
+- [ ] Have you included a short summary of what the PR does, a description of the problem being solved and the pros/cons of the approach?
+- [ ] Does it make sense for this PR to be of this size? If the PR is large, consider breaking it down into smaller incremental PRs.
+- Everyone merges their own PRs. Respond to reviewer feedback before merging. Try to avoid taking feedback personally.
```

Comment:
This should also be a hidden comment, since it's not relevant to the content of the PR.